app/src/main/cpp/skyline/vfs/os_filesystem.h

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include "filesystem.h"
+
+namespace skyline::vfs {
+    /**
+     * @brief The OsFileSystem class abstracts an OS folder with the vfs::FileSystem api
+     */
+    class OsFileSystem : public FileSystem {
+      private:
+        std::string basePath; //!< The base path for filesystem operations
+
+      public:
+        OsFileSystem(std::string basePath);
+
+        bool CreateFile(std::string path, size_t size);
+
+        bool CreateDirectory(std::string path, bool parents);
```

Comment:
It is quite ambigious what `parents` here means, a comment would be appreciated

Reply:
It's a virtual function override :)

Reply:
I see, my bad